src/components/Navbar.jsx

```diff
@@ -101,11 +92,13 @@ const Navbar = () => {
             <li>
               <NavLink
                 to="/ContributorsPage"
```

Comment:
change to the `contributorspage`

```suggestion
                to="/contributorspage"
```